package.json

```diff
@@ -4,7 +4,7 @@
   "license": "MIT",
   "scripts": {
     "format": "prettier --write . && yarn lint:fix",
-    "lint": "eslint .",
+    "lint": "eslint --no-eslintrc --config .eslintrc.js .",
```

Comment:
@brookslybrand This was it! This ignores all `.eslintrc.cjs` files from the new v2/vite examples :)